src/og/renderer/Renderer.js

```diff
@@ -508,22 +517,24 @@ class Renderer {
         }
     }
 
-    resize() {
+    _resizeStart() {
         let c = this.handler.canvas;
-        this.activeCamera.setAspectRatio(c.width / c.height);
-        this.sceneFramebuffer.setSize(c.width, c.height);
 
-        this.blitFramebuffer &&
-        this.blitFramebuffer.setSize(c.width, c.height, true);
+        this.activeCamera.setAspectRatio(c.width / c.height);
+        this.sceneFramebuffer.setSize(c.width * 0.5, c.height * 0.5);
+        this.blitFramebuffer && this.blitFramebuffer.setSize(c.width * 0.5, c.height * 0.5, true);
+    }
 
```

Comment:
Come on! And what does splitting the resize pipeline give? why exactly 20 seconds? maybe instead of 20 seconds requestAnimationFrame? can't wait to run  it in devtools)

Reply:
Changed to 500, you were right! Next time will check observers.